### PR TITLE
feat(developer-manual) User management section

### PR DESCRIPTION
**Pull request by aurelien-baudet**

### CODE REVIEW

**Comment by aurelien-baudet** [comment]
Preview : http://ci.zpush.io:28702/job/ZetaPush%20Github/job/documentation/view/change-requests/job/PR-124/Generated_20documentation/index.html#_standard_user_workflow

File: src/docs/asciidoc/common/cloud-services/standard-user-workflow/quick-start.adoc [hunk 1]
```diff
@@ -28,98 +28,208 @@ $ npm install --save @zetapush/user-management
...
+for example two _cloud functions_ named `hello` in two different 
+_cloud services_ that are exposed. So on the client side, the name
+is used to specify which _cloud service_ to use when we call
+the _cloud function_ `hello`.
+====
+
+.Listen to HTTP requests
+[NOTE]
+====
+ZetaPush uses a bidirectional connection to interact between client and _cloud services_ (and vice versa).
```
[the remark this answers is not in the document]
Comment (aurelien-baudet) [reply src/docs/asciidoc/common/cloud-services/standard-user-workflow/quick-start.adoc]: Yes ;)

File: src/docs/asciidoc/common/cloud-services/standard-user-workflow/quick-start.adoc [hunk 2]
```diff
@@ -28,98 +28,208 @@ $ npm install --save @zetapush/user-management
...
+The code is ready but in order to adapt the workflow to your needs, some configuration properties are required. 
+To configure those properties, you need to create a file named `application.json` at the root of your application:
 
 .application.json
 [source,json,linenums]
 ----
 {
   "email": {
-    "from": "celtia@zetapush.com" <1>
+    "from": "no-reply@your-application-name.com"    // <1>
```
[the remark this answers is not in the document]
Comment (aurelien-baudet) [reply src/docs/asciidoc/common/cloud-services/standard-user-workflow/quick-start.adoc]: I agree. What do you think about:
<1> As an email is sent to the end user for confirming its account, you need to configure the email sender. The email sender address may not be totally what you want and may depend on your email provider. For the example, we use a testing service that allows any email address as email sender.

[the remark this answers is not in the document]
Comment (damienld22) [reply src/docs/asciidoc/common/cloud-services/standard-user-workflow/quick-start.adoc]: It's ok for me

**Pull request merged**